.github/workflows/release-stg.yaml

```diff
@@ -0,0 +1,18 @@
+name: release-stg 
+
+on:
+  workflow_dispatch:
```

Comment:
Do we want this on a cron too?

Reply:
I would say yes because all of our other staging workflows run on a cron.

Reply:
yeah we need this to run nightly